supervisor/install/install_windows.go

```diff
@@ -17,9 +17,88 @@
 
 package install
 
-import "errors"
+import (
+	"fmt"
+	"log"
 
-// Install installs service for linux. Not implemented yet.
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/eventlog"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+const serviceName = "MystSupervisor"
+
+// Install installs service for Windows.
 func Install(options Options) error {
-	return errors.New("not implemented")
+	m, err := mgr.Connect()
+	if err != nil {
+		return err
+	}
+	defer m.Disconnect()
+
+	config := mgr.Config{
+		ServiceType:  windows.SERVICE_WIN32_OWN_PROCESS,
+		StartType:    mgr.StartAutomatic,
+		ErrorControl: mgr.ErrorNormal,
+		DisplayName:  "MystSupervisor Service",
+		Description:  "Mysterium Network dApp supervisor service is responsible for manages network configurations",
+	}
+
+	if err := uninstallService(m, serviceName); err != nil {
+		log.Printf("Failed to remove service: %v", err)
+	}
+
+	if err := installAndStartService(m, serviceName, options, config); err != nil {
+		return fmt.Errorf("could not install and run service: %w", err)
+	}
+	return nil
+}
+
+func installAndStartService(m *mgr.Mgr, name string, options Options, config mgr.Config) error {
+	s, err := m.OpenService(name)
+	if err == nil {
```

Comment:
It is more idiomatic to compare against error
```suggestion
	if err != nil {
```